Library/Formula/llvm.rb

```diff
@@ -94,7 +94,7 @@ class Llvm < Formula
   deprecated_option "rtti" => "with-rtti"
   deprecated_option "disable-assertions" => "without-assertions"
 
-  if MacOS.version <= :snow_leopard
+  if MacOS.version <= :snow_leopard || build.with?("python")
```

Comment:
`depends_on :python` is [special](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/requirements/python_requirement.rb).